### ISSUE TITLE
Content draft

### ISSUE THREAD

**Issue by isysd** [deginner/gitguild_website#1]
Write draft content for the git guild project website. Use the [standard template](https://github.com/deginner/project-standards/blob/master/standards/website.md), and draft papers as content.

https://bitbucket.org/snippets/deginner/6rL79 (public)
https://bitbucket.org/snippets/deginner/6bLKj (private)

**Comment by isysd** [comment]
# Git Guild Bounty Program

| Total | Dev               | Sales | Reporter | Tester(s) | Maintainer | Documenter |
|--------|------------------|---------|-------------|------------|-----------------|----------------|
|N/A    | assignee(s)   | N/A  | @isysd | @isysd, @d3brouille | @isysd, @d3brouille  | @d3brouille |
| $400.00 | $366.67 | $0.00 | $33.33 | $133.33 | $33.33 |

**Comment by ChrisSavoie** [comment]
@isysd @glenkow How relevant are these documents today? Still up-to-date? This was before the split from the other brand?

**Comment by glenkow** [comment]
Here's a link to the wireframe for the home page. https://glenkowalski.mybalsamiq.com/mockups/4827885.png?key=4f59c2cd9c120d76d36be8150c55267c804152ef

**Comment by isysd** [comment]
Looking sexy @glenkow :+1: Seems like we might need a paired down version of this for the rapid fire MVP launch, no? We'll have to work up content over time.

**Comment by glenkow** [comment]
Yes, we can pair this down for now. The place holders are valid, we can add coolness like floating menus and sliders down the line.

**Comment by glenkow** [comment]
Wire frame for About page. The "Section" text each corresponds to an existing MD file. I will modify the copy in those directly.

https://glenkowalski.mybalsamiq.com/mockups/4846630.png?key=d80b6751dc154e07f54914a4ef89cb40045dfacd

**Comment by glenkow** [comment]
I polished the 4 sections referenced above and I think they have more clarity now. Read them and let me know what you think. They might still need some enhancing, and I can interview @isysd about that to make sure we are all on the same page.
